ansible/roles/ocp4-workload-cnvlab/tasks/remove_workload.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Implement your workload removal tasks here
+# ------------------------------------------
+
+- name: Remove the CNV workspace
+  command: oc delete project workbook
```

Comment:
Better would be `k8s` with state *absent*

Reply:
Great, thanks!